### PR TITLE
[IOPAE-Vehicles]: UI fixes

### PR DESCRIPTION
**Pull request by PierpaoloPirasSp**
#### List of Changes

Some fix reported by design:
[IOPAE-1628](https://pagopa.atlassian.net/jira/software/c/projects/IOPAE/boards/289?assignee=62bd660cde439ce00ee4c373&selectedIssue=IOPAE-1628)
[IOPAE-1629](https://pagopa.atlassian.net/jira/software/c/projects/IOPAE/boards/289?assignee=62bd660cde439ce00ee4c373&selectedIssue=IOPAE-1629)
[IOPAE-1631](https://pagopa.atlassian.net/jira/software/c/projects/IOPAE/boards/289?assignee=62bd660cde439ce00ee4c373&selectedIssue=IOPAE-1631)

#### Screenshots (if appropriate):

1628:
 
Screenshot 2024-12-06 alle 17 18 08

1629:

Screenshot 2024-12-06 alle 17 22 39
Screenshot 2024-12-06 alle 17 22 08
Screenshot 2024-12-06 alle 17 25 38

1631:

Screenshot 2024-12-06 alle 17 20 42

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

[IOPAE-1628]: https://pagopa.atlassian.net/browse/IOPAE-1628?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[IOPAE-1629]: https://pagopa.atlassian.net/browse/IOPAE-1629?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[IOPAE-1630]: https://pagopa.atlassian.net/browse/IOPAE-1630?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[IOPAE-1631]: https://pagopa.atlassian.net/browse/IOPAE-1631?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/ui/lib/components/dialog/__tests__/__snapshots__/Dialog.test.tsx.snap [hunk 1]
```diff
@@ -82,7 +82,7 @@ exports[`Test Dialog Component > Should match the snapshot with all props 1`] =
                 >
                   <path
                     d="M0 24C0 10.7452 10.7452 0 24 0C37.2548 0 48 10.7452 48 24C48 37.2548 37.2548 48 24 48C10.7452 48 0 37.2548 0 24Z"
-                    fill="currentColor"
+                    fill="#E6F9EF"
```
Comment (giuseppedipinto) [inline packages/ui/lib/components/dialog/__tests__/__snapshots__/Dialog.test.tsx.snap:85]: why can't we get the value to work dynamically anymore? Is there some impediment or blocking issue?

File: packages/ui/lib/components/icon/svg/IconError.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ const IconError = (props: SvgIconProps) => (
   <SvgIcon viewBox="0 0 48 48" {...props}>
     <path
       d="M0 24C0 10.7452 10.7452 0 24 0C37.2548 0 48 10.7452 48 24C48 37.2548 37.2548 48 24 48C10.7452 48 0 37.2548 0 24Z"
-      fill="currentColor"
+      fill="#FCE8EB"
```
Comment (giuseppedipinto) [inline packages/ui/lib/components/icon/svg/IconError.tsx:7]: why can't we get the value to work dynamically anymore? Is there some impediment or blocking issue?

File: packages/ui/lib/components/icon/svg/IconSuccess.tsx [hunk 3]
```diff
@@ -4,7 +4,7 @@ const IconSuccess = (props: SvgIconProps) => (
   <SvgIcon viewBox="0 0 48 48" {...props}>
     <path
       d="M0 24C0 10.7452 10.7452 0 24 0C37.2548 0 48 10.7452 48 24C48 37.2548 37.2548 48 24 48C10.7452 48 0 37.2548 0 24Z"
-      fill="currentColor"
+      fill="#E6F9EF"
```
Comment (giuseppedipinto) [inline packages/ui/lib/components/icon/svg/IconSuccess.tsx:7]: why can't we get the value to work dynamically anymore? Is there some impediment or blocking issue?

File: packages/ui/lib/components/icon/svg/IconWarning.tsx [hunk 4]
```diff
@@ -4,7 +4,7 @@ const IconWarning = (props: SvgIconProps) => (
   <SvgIcon viewBox="0 0 48 48" {...props}>
     <path
       d="M0 24C0 10.7452 10.7452 0 24 0C37.2548 0 48 10.7452 48 24C48 37.2548 37.2548 48 24 48C10.7452 48 0 37.2548 0 24Z"
-      fill="currentColor"
+      fill="#F6E4C8"
```
Comment (giuseppedipinto) [inline packages/ui/lib/components/icon/svg/IconWarning.tsx:7]: why can't we get the value to work dynamically anymore? Is there some impediment or blocking issue?

**Review by giuseppedipinto — CHANGES_REQUESTED** [review]

File: packages/ui/lib/components/icon/svg/IconWarning.tsx (hunk 4 above)
Comment (PierpaoloPirasSp) [reply packages/ui/lib/components/icon/svg/IconWarning.tsx:7]: In these three cases, the icons have a shadow that, when assigned with currentColor as happens a bit further below, results in the icon's shape not being displayed correctly.

Screenshot 2024-12-09 alle 12 55 46


I haven't found an alternative at the moment.

**Pull request merged**